src/iOS/Xamarin.iOS/Samples/Augmented Reality/ViewHiddenInfrastructureAR/readme.md

```diff
@@ -0,0 +1,60 @@
+
+1. See the 'Sketch on map' sample to learn how to use the sketch editor for creating graphics.
+2. When you start the AR visualization experience, create and show the `ARSceneView`.
+3. Access the `Scene` property of the AR View and set the space effect `SpaceEffect.None` and the atmosphere effect to `AtmosphereEffect.None`.
+4. Create an elevation source and add it to the scene's base surface. Set the navigation constraint to `NavigationConstraint.None` to allow going underground if needed.
+5. Configure a graphics overlay and renderer for showing the drawn pipes. This sample uses a `SolidStrokeSymbolLayer` with a `MultilayerPolylineSymbol` to draw the pipes as tubes. Add the drawn pipes to the overlay.
+6. Configure the calibration experience.
+    * When in roaming (continuous location update) mode, only heading calibration should be enabled. In continuous update mode, the user's calibration is overwritten by sensor-based values every second.
+    * When in local mode, the user needs to be able to adjust the heading, altitude, and position; position adjustment is achieved by panning.
+    * This sample uses a basemap as a reference during calibration; consider how you will support your user's calibration efforts. A basemap-oriented approach won't work indoors or in areas without readily visible, unchanging features like roads.
+
+## Relevant API
+
+* ARSceneView
+* GraphicsOverlay
+* MultilayerPolylineSymbol
+* SketchEditor
+* SolidStrokeSymbolLayer
+* Surface
+
+## About the data
+
+This sample uses Esri's [world elevation service](https://elevation3d.arcgis.com/arcgis/rest/services/WorldElevation3D/Terrain3D/ImageServer) to ensure that the infrastructure you create is accurately placed beneath the ground.
+
+Real-scale AR relies on having data in real-world locations near the user. It isn't practical to provide pre-made data like other Runtime samples, so you must draw your own nearby sample 'infrastructure' prior to starting the AR experience.
+
+## Additional information
+
+This sample requires a device that is compatible with ARKit 1 on iOS or ARCore 1.8 on Android.
+
+Note that unlike other scene samples, a basemap isn't shown most of the time, because the real world provides the context. Only while calibrating is the basemap displayed at 50% opacity, to give the user a visual reference to compare to.
+
+You may notice that pipes you draw underground appear to float more than you would expect. That floating is a normal result of the parallax effect that looks unnatural because you're not used to being able to see underground/obscured objects. Compare the behavior of underground pipes with equivalent pipes drawn above the surface - the behavior is the same, but probably feels more natural above ground because you see similar scenes day-to-day (e.g. utility wires).
+
+**Real-scale AR** is one of three main patterns for working with geographic information in augmented reality. See  [Agumented reality]() in the guide for more information.
```

Comment:
Doc link: https://developers.arcgis.com/net/latest/ios/guide/display-scenes-in-augmented-reality.htm